library/scrollphathd/api/run_http_api.py

```diff
@@ -0,0 +1,24 @@
+from .http import scrollphathd_blueprint
+from Flask import Flask
```

Comment:
`s/Flask/flask`

I'm inclined to stuff all of this into, or at least call it from, `scrollphathd.api.http` so `python -m scrollphathd.api.http` works.

Reply:
yeah, I think PyCharm was having an off day here. Didn't think I had flask installed but for some reason thought Flask was installed? 🤷‍♀️ 

Sure, I'll move it :)